systemtests/src/main/java/io/enmasse/systemtest/platform/Minikube.java

```diff
@@ -34,10 +34,11 @@ protected Minikube(Environment environment) {
             OkHttpClient httpClient = HttpClientUtils.createHttpClient(config);
             // Workaround https://github.com/square/okhttp/issues/3146
             httpClient = httpClient.newBuilder()
-                    .protocols(Collections.singletonList(Protocol.HTTP_1_1))
+                    .protocols(Collections.singletonList(Protocol.HTTP_2))
```

Comment:
It was this change #2938 that switched to HTTP_1_1 to avoid a OkHttp client bug (https://github.com/square/okhttp/issues/3146).  The defect has a fixed milestone of v4 (and maybe 3.12.8 judging by comments in the thread).  We still pickup 3.12.6 transitively from fabric8.    If we want to switch to HTTP_2, then we probably need to override okhttp too to pickup 3.12.8

Reply:
@k-wall so do you think that we can remove this workaround?